File: src/main/java/ar/edu/unq/spring/modelo/exception/NombreDePersonajeRepetido.java
package ar.edu.unq.spring.modelo.exception;

public class NombreDePersonajeRepetido extends RuntimeException {

    private static final long serialVersionUID = 1L;

    public NombreDePersonajeRepetido(String message) {
        super("El nombre de personaje [" + message + "] ya esta siendo utilizado y no puede volver a crearse");
    }
}

File: src/main/kotlin/ar/edu/unq/spring/modelo/exception/NombreDePersonajeRepetido.kt
package ar.edu.unq.spring.modelo.exception

import ar.edu.unq.spring.modelo.Item
import ar.edu.unq.spring.modelo.Personaje

class NombreDePersonajeRepetido(private val nombre: String) : RuntimeException() {

    override val message: String?
        get() = "El nombre de personaje [$nombre] ya esta siendo utilizado y no puede volver a crearse"

    companion object {

        private val serialVersionUID = 1L
    }

}
